docs/setup-robusta/gitops/argocd.rst

```diff
@@ -120,8 +120,9 @@ Finally, run ``robusta logs`` from your cli and make sure there is no error.
 
     On some Robusta versions, the sync might fail with ``CustomResourceDefinition.apiextensions.k8s.io “prometheuses.monitoring.coreos.com” is invalid: metadata.annotations: Too long: must have at most 262144 bytes``.
 
-    To solve it, use the workaround proposed `here <https://github.com/prometheus-community/helm-charts/issues/1500#issuecomment-1132907207>`_
+    To solve it, use the workaround proposed :ref:`here <CRD Issues>`.
```

Comment:
Wrong fix. This is the server-side apply one.

Reply:
https://blog.ediri.io/kube-prometheus-stack-and-argocd-25-server-side-apply-to-the-rescue

Reply:
Fixed at the reference

Reply:
Can you link it directly from here to save the user another click?